mne/viz/montage.py

```diff
@@ -26,6 +26,11 @@ def plot_montage(montage, scale_factor=20, show_names=True, kind='topomap',
         Show figure if True.
     %(sphere_topomap_auto)s
     %(verbose)s
+    axes : instance of Axes | instance of Axes3D | None
+        Axes to draw the sensors to. If ``kind='3d'``, axes must be an instance
+        of Axes3D. If None (default), a new axes will be created.
+
+        .. versionadded:: 1.4.0
```

Comment:
Ideally this would become

```suggestion
    %(axes_plot_sensors)s

        .. versionadded:: 1.4.0
```
and then a similar change here:
https://github.com/mne-tools/mne-python/blob/ea88c55b183b5ed5aec793737caf30d750ac999e/mne/viz/utils.py#L939-L943
... and then the duplicated docstring gets added to the docdict in between these two entries:
https://github.com/mne-tools/mne-python/blob/ea88c55b183b5ed5aec793737caf30d750ac999e/mne/utils/docs.py#L283-L285

If that sounds doable, please give it a try; if not LMK and I can push a commit that does this.